BuildCustomFunctions.ps1

```diff
@@ -47,13 +47,30 @@ Try
             $NewCommandName = $CommandName.Replace($ConfigPrefix, $ConfigCustomFunctionPrefix)
             # Get content from sdk function
             $CommandFilePath = $Command.ScriptBlock.File
+            # TODO: is this the fix
+            # $CommandFilePath = ($Command.ScriptBlock.File -replace "ProxyCmdletDefinitions", $CommandName)
+            write-host $CommandFilePath
```

Comment:
TODO: Write out a line stating that the command is building from the $CommandFilePath definition file